ArcGISRuntimeSDKQt_QMLSamples/Features/FeatureLayer_DefinitionExpression/FeatureLayer_DefinitionExpression.qml

```diff
@@ -77,22 +77,62 @@ Rectangle {
         }
         spacing: 5
 
+        Label {
+
+            text: "Current feature count: " + updateFeatureCount();
+        }
         // button to apply a definition expression
         Button {
             text: "Apply Expression"
+            width: 200
             enabled: featureTable.loadStatus === Enums.LoadStatusLoaded
             onClicked: {
                 featureLayer.definitionExpression = "req_Type = \'Tree Maintenance or Damage\'"
             }
         }
+        // button to apply display filter
+        Button {
+            text: "Apply Display Filter"
+            width: 200
+            enabled: featureTable.loadStatus === Enums.LoadStatusLoaded
+            onClicked: {
+                if (featureLayer.loadStatus === Enums.LoadStatusLoaded)
+                {
+                    var displayFilter = ArcGISRuntimeEnvironment.createObject("DisplayFilter", {name: "Damaged Trees", filterId: "Damaged Trees", whereClause: "req_Type = \'Tree Maintenance or Damage\'"});
+                    var displayFilterDefintionVar = ArcGISRuntimeEnvironment.createObject("ManualDisplayFilterDefinition");
+                    displayFilterDefintionVar.availableFilters.append(displayFilter);
+                    displayFilterDefintionVar.activeFilter = displayFilter;
+
+                    featureLayer.displayFilterDefinition = displayFilterDefintionVar;
+                }
+            }
+        }
 
         // button to reset the definition expression
         Button {
             text: "Reset"
+            width: 200
             enabled: featureTable.loadStatus === Enums.LoadStatusLoaded
             onClicked: {
                 featureLayer.definitionExpression = "";
+
+                var displayFilter = ArcGISRuntimeEnvironment.createObject("DisplayFilter", {id: "No Filter", whereClause: "1=1"});
+                var displayFilterDefintionVar = ArcGISRuntimeEnvironment.createObject("ManualDisplayFilterDefinition");
+                displayFilterDefintionVar.activeFilter =  displayFilter;
+                displayFilterDefintionVar.availableFilters.append(displayFilter);
+                featureLayer.displayFilterDefinition = displayFilterDefintionVar;
             }
         }
     }
+
+    function updateFeatureCount()
+    {
+        const queryParams = ArcGISRuntimeEnvironment.createObject(
+                              "QueryParameters", {
+                                  "geometry": mv.currentViewpointExtent.extent
+                              });
+
+        featureTable.queryFeatureCount(queryParams);
+        return featureTable.queryFeatureCountResult;
```

Comment:
❌ 
This will not work, but luckily it's even easier to do.

I would make this method return void since the return value here won't by in sync with the call. `queryFeatureCount` is async, so you're likely to get a stale or invalid number returned here every time.

You will also need a minor update so that this is called after some operation. The binding expression for the label will only call this once.